internal/services/storage/storage_management_policy_data_source.go

```diff
@@ -94,29 +94,45 @@ func dataSourceStorageManagementPolicy() *pluginsdk.Resource {
 													Type:     pluginsdk.TypeInt,
 													Computed: true,
 												},
-												"tier_to_archive_after_days_since_modification_greater_than": {
+												"tier_to_cool_after_days_since_last_access_time_greater_than": {
 													Type:     pluginsdk.TypeInt,
 													Computed: true,
 												},
-												"delete_after_days_since_modification_greater_than": {
+												"auto_tier_to_hot_from_cool_enabled": {
+													Type:     pluginsdk.TypeBool,
+													Optional: true,
+												},
+												"tier_to_cool_after_days_since_creation_greater_than": {
 													Type:     pluginsdk.TypeInt,
-													Computed: true,
+													Optional: true,
+												},
+												"tier_to_archive_after_days_since_modification_greater_than": {
+													Type:     pluginsdk.TypeInt,
+													Optional: true,
 												},
 												"tier_to_archive_after_days_since_last_access_time_greater_than": {
 													Type:     pluginsdk.TypeInt,
-													Computed: true,
+													Optional: true,
 												},
 												"tier_to_archive_after_days_since_last_tier_change_greater_than": {
 													Type:     pluginsdk.TypeInt,
-													Computed: true,
+													Optional: true,
+												},
+												"tier_to_archive_after_days_since_creation_greater_than": {
+													Type:     pluginsdk.TypeInt,
+													Optional: true,
+												},
+												"delete_after_days_since_modification_greater_than": {
+													Type:     pluginsdk.TypeInt,
+													Optional: true,
 												},
 												"delete_after_days_since_last_access_time_greater_than": {
 													Type:     pluginsdk.TypeInt,
-													Computed: true,
+													Optional: true,
 												},
-												"tier_to_cool_after_days_since_last_access_time_greater_than": {
+												"delete_after_days_since_creation_greater_than": {
 													Type:     pluginsdk.TypeInt,
-													Computed: true,
+													Optional: true,
```

Comment:
this is a Data Source, so all of these should be `Computed: true` (without `Optional: true`)?

Reply:
@tombuildsstuff Yes, thank you for catching this!